### PR TITLE
Add Ansible playbooks for non-image environments

### LINKED ISSUE #5: [2] Provision machines to be ready for k8s installation
Since we won't use the pre baked `ck8s-image` the machines will need to install all the requirements before the k8s installation. This will include.
* Docker
* kubectl
* kubeadm
* kubelet
* disable swap.

And the load-balancer and nfs will need ha-proxy and nfs-server installed
use https://github.com/elastisys/ck8s-base-vm for insparation on whats needed.

This should be done in ansible if nobody has any better ideas.

### PR DESCRIPTION
**Pull request by simonklb**
Fixes https://github.com/elastisys/ck8s-cluster/issues/5

Tested with:

```
% cd vagrant
% vagrant up
% ansible-playbook -i inventory.ini -e @../variables.json ../ansible/provision.yaml
% ansible-playbook -i inventory.ini -e kubeconfig_path=$(pwd)/kubeconfig ../../ck8s-cluster/ansible/deploy-kubernetes.yml
% kubectl --kubeconfig ./kubeconfig get node
NAME      STATUS   ROLES    AGE   VERSION
master1   Ready    master   80s   v1.17.11
worker1   Ready    <none>   44s   v1.17.11
```

### CODE REVIEW

**Review by Ajarmar — APPROVED** [review]
LGTM (although I'm not familiar with which details are particularly important to look at here so I may have missed something - should probably wait for another review).

Provided testing instructions didn't work with ansible 2.5.1 - not really a problem IMO, especially since the version it's tested with is mentioned in the readme, but might be good to note since 2.5.1 seems to be the version installed when running the ck8s setup.

**Review by viktor-f — COMMENTED** [review]
I have not had the time to test this yet. But it looks good so I would be ok with merging it.

**Comment by simonklb** [comment]
> LGTM (although I'm not familiar with which details are particularly important to look at here so I may have missed something - should probably wait for another review).
> 
> Provided testing instructions didn't work with ansible 2.5.1 - not really a problem IMO, especially since the version it's tested with is mentioned in the readme, but might be good to note since 2.5.1 seems to be the version installed when running the ck8s setup.

Thanks for the feedback! I believe there is some work going into upgrading Ansible in the other projects as well. :)

**Pull request merged**
